utility/src/main/java/org/oppia/android/util/logging/firebase/BUILD.bazel

```diff
@@ -0,0 +1,79 @@
+"""
+General purpose utilities for firebase logging.
```

Comment:
This might be a bit clearer: ``Logging utility implementations backed by Firebase.`` (since that's what this package actually is).

Reply:
Done